scan-website.sh

```diff
@@ -1,15 +1,15 @@
 website_url=$1
 scan_text=$2
-status_code=$(curl --silent --head $website_url | head -1 | awk '{print $2}')
+status_code=$(curl --silent --location --head --output /dev/null --write-out "%{http_code}" "${website_url}")
 
-if [ "$status_code" != "200" ]
-then
-    echo "Website is unreachable"
+if [ "${status_code}" != "200" ]; then
+    echo "Website is unreachable (${status_code})" >&2
     exit 1
 fi
 
-if [ -z $(curl --silent $website_url | grep -i "$2") ]
-then
-    echo "Input text was not found on website."
+if ! $(curl --silent --location "{$website_url}" | grep --quiet --ignore-case "${scan_text}"); then
```

Comment:
Actually doesn't need the surrounding `$( )` subshell interpolation and should be removed:

`if ! curl --silent --location "{$website_url}" | grep --quiet --ignore-case "${scan_text}"; then`